src/lib/util/jedparse.h

```diff
@@ -29,7 +29,10 @@
 #define JEDERR_BAD_XMIT_SUM     2
 #define JEDERR_BAD_FUSE_SUM     3
 
-
+enum fileformats {
+  MAXLOADER,
+  DATAIO
+};
```

Comment:
You’ve put three unacceptably generic names in the global scope.

Reply:
Yes, I will fix it somehow.